config/config.go

```diff
@@ -40,12 +40,18 @@ type Configuration struct {
 	// Can be provided by FromEnv() via the environment variable named JAEGER_SERVICE_NAME
 	ServiceName string `yaml:"serviceName"`
 
-	// Disabled can be provided by FromEnv() via the environment variable named JAEGER_DISABLED
+	// Disabled makes the config return opentracing.NoopTracer.
+	// Value can be provided by FromEnv() via the environment variable named JAEGER_DISABLED.
 	Disabled bool `yaml:"disabled"`
 
-	// RPCMetrics can be provided by FromEnv() via the environment variable named JAEGER_RPC_METRICS
+	// RPCMetrics enables generations of RPC metrics (requires metrics factory to be provided).
+	// Value can be provided by FromEnv() via the environment variable named JAEGER_RPC_METRICS
```

Comment:
No good deed goes unpunished :-) Would it be easy to add the possible values to this property?

Reply:
Scratch that... Just realized this is also a boolean. Thought it would be the factory to use.